drivers/i2s/i2s_ll_stm32.c

```diff
@@ -848,8 +848,8 @@ static struct device *get_dev_from_tx_dma_channel(u32_t dma_channel)
 		.block_count = 1,					\
 		.dma_slot = DT_I2S_##index##_DMA_SLOT_##dir_cap,	\
 		.channel_direction = PERIPHERAL_TO_MEMORY,		\
-		.source_data_size = 1,  /* 16bit default */		\
```

Comment:
Shouldn't it logically be done after new driver introduction?

Reply:
Yes, this should be after the new driver introduction.